src/Service/CloudFormation/CHANGELOG.md

```diff
@@ -1,13 +1,13 @@
 # Change Log
 
+## NOT RELEASED
+
 ## 0.5.0
 
 ### Deprecation
```

Comment:
```suggestion
### Changed
```

Reply:
Changed (for all services but thoses tagged 1.0)